.buildkite/A100_pipeline/pipeline.yml

```
@@ -1,6 +1,6 @@
 agents:
   queue: clima
-  modules: climacommon/2024_04_05
+  modules: julia/1.10.5 cuda/julia-pref nsight-systems/2024.6.1
```

```suggestion
  modules: climacommon/2024_10_08
```
(or 09 if you want julia 1.11)

---

You can also keep it explicit, except that `cuda/julia-pref` is not a valid module

---

Moved to `climacommon/2024_10_08`